Perlang.Interpreter/Immutability/ImmutabilityValidator.cs

```diff
@@ -22,14 +22,6 @@ internal class ImmutabilityValidator : VisitorBase
             new ImmutabilityValidator(immutabilityValidationErrorCallback, getVariableOrFunctionBinding).Visit(statements);
         }
 
-        public static void Validate(
-            Expr expr,
-            Action<ValidationError> immutabilityValidationErrorCallback,
-            Func<Expr, Binding?> getVariableOrFunctionBinding)
-        {
-            new ImmutabilityValidator(immutabilityValidationErrorCallback, getVariableOrFunctionBinding).Visit(expr);
-        }
-
```

Comment:
:tada: No need for this any more now since we unified all immutability validation to use the overload which takes a `List<Stmt>` parameter.